### ISSUE TITLE
[拓展章节] Graph + MLSys

### ISSUE THREAD

**Issue by hansen7** [openmlsys/openmlsys-zh#231]

**Comment by luomai** [comment]
可以的。我们大概花2个月左右时间写这章。先在这里讨论下章节的内容。

**Comment by eedalong** [comment]
GNN应用
 计算范式: Message Passing 
 编程范式：Tensor-Centric(PyG), Graph-Centric(DGL), Vertex-Centric(MindSpore-GraphLearning)
 整图学习系统： RoC
 基于采样子图Mini-Batch学习系统： PyG， DGL

**Comment by eedalong** [comment]
@luomai @hansen7  本章节需要覆盖的一些内容的proposal～

**Comment by yzh119** [comment]
I don't agree that there are differences between the paradigms between DGL, PyG, and Seastar. All of them adopt vertice/edge-centric DSL. The only differences are how is the DSL compiled and executed, and the methodology converges as time went by.

I encourage paying more attention to distributed training of GNNs: how to optimize pipelining, networking, sampling, etc.

**Comment by luomai** [comment]
@yzh119 谢谢建议。图学习系统这块我们还没有开始梳理出来。很欢迎提建议和参与。我们最近在忙第一版的发布。忙完这阵就会开始思考图学习系统的章节了。

**Comment by yofufufufu** [comment]
各位老师好，想请问图学习这一块进展如何了？对Graph+MLSys感兴趣，非常期待！
